### ISSUE TITLE
Add feature to add and edit Pull Request comments in the Editor

### ISSUE THREAD

**Issue by ursmeili** [microsoft/azure-devops-intellij#560]
With this plugin, I can only view pull requests, but I cannot add or edit comments to the pull request right in the IDE. This would be a great feature.

**Comment by Jonatha1983** [comment]
@ursmeili I just release a new AZD plugin it is based on the Jetbrains Github bundle plugin and it should be available in the marketplace in 1-2 days
I am sure there will be some bugs at the start and I will love to collect them so can fix and continue improving them. 
this is the [plugin page](https://plugins.jetbrains.com/plugin/22319-azd) again should be available in 1-2 days

Cheers

**Comment by ursmeili** [comment]
Hi,
unfortunately it doesn't seem compatible with Rider, only with IntelliJ and
Android Studio. If you can make it compatible with Rider, I definitely give
it a try.
Cheers

**Comment by Jonatha1983** [comment]
@ursmeili  yes I just uploaded a new version that should be compatible with all JetBrains IDE - it's in Jetbrains marketplace review now - hopefully will be available soon. 

Thanks

**Comment by ursmeili** [comment]
sadly I cannot use your tool, since you seem to support only Azure DevOps
in the cloud, not on-premise. What a shame.

**Comment by Jonatha1983** [comment]
Yes currently only azure devops service 🤷🏼‍♂️
